files/en-us/web/css/anchor/index.md

```diff
@@ -404,7 +444,7 @@ The anchors are each given a different {{cssxref("anchor-name")}} values, a {{cs
 }
 ```
 
-The anchor-positioned element is associated with one anchor via its {{cssxref("position-anchor")}} property; its `position` is set to `fixed`. It is positioned relative to two anchors by setting two different `<anchor-name>` values with the `anchor()` functions set on its inset properties. In this case, we used {{cssxref("percentage")}} values for the `<anchor-side>` parameter, specifying the distance from the start of the axis of the inset property on which the function is set.
+The anchor-positioned element, with its `position` is set to `fixed`, is associated with one anchor via its {{cssxref("position-anchor")}} property. It is positioned relative to two anchors by including two different `<anchor-name>` values with the `anchor()` functions set on its inset properties. In this case, we used {{cssxref("percentage")}} values for the `<anchor-side>` parameter, specifying the distance from the start of the axis of the inset property on which the function is set.
```

Comment:
I am being pedantic, but the element is only positioned once it has `position` set, so put it first.